src/infrastructure/persistance/item_api_v1_repository.py

```diff
@@ -41,12 +42,12 @@ def post(self, item: dict) -> Item:
         )
         return Item(**response.json()["result"])
 
-    def patch(self, id: int, item: dict) -> dict:
+    def patch(self, id: int, patch_info: PatchInfo) -> PatchInfo:
```

Comment:
A question to meditate...

I'm not sure if `patch_info` it's a good name for this variable or the DTO.

What do you think?

🤔

Reply:
tbh I wasn't sold on the name. I tried PatchData or smth like that but wasn't sure either. Wdyt?